history/cultures/dalaranian.txt

```diff
@@ -38,6 +38,10 @@
 	#
 	discover_innovation = innovation_elite_ranks
 	#
+	discover_innovation = innovation_guerilla_warfare
```

Comment:
What? They don't live in forests or jungles.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/136604006-f8355f80-1957-4f75-ae5c-bd94b2d07fb5.png)

</details>

Your commits need more info.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/136604221-4d187778-69bf-450c-88f2-669d23a3ef40.png)

</details>

You don't say what you change and why.

Reply:
Hmm my bad, someone left a comment that they were fully eligible for it, but didn't get it in their history and I took their word for it. I will remove it.